### PR TITLE
{chem}[iccifort/2020.4.304] BEEF v0.1.1

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/4c618208503d54d90519a6ff5cfc6039 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/7132a7aefed088e883106ebade49167d for a full test report.

File: easybuild/easyconfigs/b/BEEF/BEEF-0.1.1-iccifort-2020.4.304.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'BEEF'
+version = '0.1.1'
+
+homepage = 'svn://suncatls1.slac.stanford.edu/beef'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/b/BEEF/BEEF-0.1.1-iccifort-2020.4.304.eb:6]: what about using https://confluence.slac.stanford.edu/display/SUNCAT/BEEF+Functional+Software ?

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/b/BEEF/BEEF-0.1.1-iccifort-2020.4.304.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/b/BEEF/BEEF-0.1.1-iccifort-2020.4.304.eb:6]: yeah I'm not sure how i overlooked that svn url in there.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/5532c0a967c525963fdde4db4885463b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/5eae7bc008bdf69418f1c2e2633fe211 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3140.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/e31ae6d1bac44c623e4672653a5f488c for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**